chapters/project_guide.rst

```diff
@@ -1,5 +1,5 @@
 #####################
-Project setup guide
+How to Set up a Jina Correctly
```

Comment:
How to Set up Jina Correctly

or 

Jina set up guide

Reply:
Good spot!